ldk/javascript/examples/react-loop/src/index.js

```diff
@@ -0,0 +1,576 @@
+import * as React from 'react';
```

Comment:
This might be a big ask but could you change this to a TypeScript `.tsx` file and add types where necessary?

Reply:
What's a typescript?

Reply:
Updated react-loop to be a typescript!